services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -713,23 +706,27 @@ public void installVerificationRule(final DatapathId dpid, final boolean isBroad
         // Don't install the unicast for OpenFlow 1.2 doesn't work properly
         if (!isBroadcast) {
             if (ofFactory.getVersion().compareTo(OF_12) > 0) {
-                logger.debug("installing unicast verification match for {}", dpid);
+                logger.debug("Installing unicast verification match for {}", dpid);
             } else {
-                logger.debug("not installing unicast verification match for {}", dpid);
+                logger.debug("Not installing unicast verification match for {}", dpid);
                 return;
             }
         }
 
-        logger.debug("installing verification rule for {}", dpid);
+        logger.debug("Installing verification rule for {}", dpid);
 
         Match match = matchVerification(sw, isBroadcast);
-        ArrayList<OFAction> actionList = new ArrayList<>(2);
+        ArrayList<OFAction> actionList = new ArrayList<>(3);
         actionList.add(actionSendToController(sw));
         actionList.add(actionSetDstMac(sw, dpidToMac(sw)));
+
+        long cookie = isBroadcast ? VERIFICATION_BROADCAST_RULE_COOKIE : VERIFICATION_UNICAST_RULE_COOKIE;
+        long meterId = installMeterForDefaultRule(sw, cookie, isBroadcast);
```

Comment:
It will produce error `meter already exist` on each switch reconnect.

(critical)